roles/gitops_configure_repo/tasks/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 
 - name: Add gitlab ssh key to argoCD
```

Comment:
gitlab in title. 

Also I think these are only needed if you have a private repo. So perhaps these should be in a block with some check that indicates that the repo is private.

Reply:
For now I only removed the reference to gitlab.